src/platform/mbed/MbedConfig.cpp

```diff
@@ -33,121 +33,296 @@
 #include <support/CodeUtils.h>
 #include <support/logging/CHIPLogging.h>
 
+#include "kvstore_global_api/kvstore_global_api.h"
+#include "platform/mbed_error.h"
+
 namespace chip {
 namespace DeviceLayer {
 namespace Internal {
 
 // *** CAUTION ***: Changing the names or namespaces of these values will *break* existing devices.
 
+// Note: An external mbed parameter could be useful so an application can put
+// chip NVS values in a single place
+#define CHIP_CONFIG_KV_STORE_PARTITION "/kv/"
```

Comment:
Does this "/kv/" parameter is not defined by mbed? Maybe we can use the compile definitions form mbed here

Reply:
The KVStore system allows an application to have several partitions. We can have a default value for CHIP: `/kv` as it is the default mbed partition. 
However in a real application, there might be a partition for CHIP, another for user data, ... To handle that case, the partition to use can be passed to CHIP through a define.

Reply:
Sure,  the partition name passed through a define will be the best.

Reply:
I created an issue for it: https://github.com/ARMmbed/connectedhomeip/issues/62